WGS_pipeline/vcf_to_depmap.wdl

```diff
@@ -32,9 +32,9 @@ task vcf_to_depmap {
         Boolean use_multi=false
         Boolean whitelist=false
         Array[String] force_keep=['oc_brca1_func_assay__class', 'oc_brca1_func_assay__score', 'clinvar_vcf_ssr', 'clinvar_vcf_clndisdbincl', 'clinvar_vcf_clnsigincl', 'clinvar_vcf_clndnincl', 'oc_oncokb__all', 'oc_oncokb__highestdiagnosticimplicationlevel', 'cgc_other_germline_mut', 'cgc_other_syndrome/disease', 'clinvar_vcf_clnsigconf', 'center', 'cosmicfusion_fusion_id', 'tumor_barcode', 'source"', 'clinvar_vcf_filter', 'clinvar_vcf_dbvarid', 'normal_barcode', 'gencode_34_ncbibuild', 'oc_oncokb_dm__highestprognosticimplicationlevel', 'strandq', 'oc_oncokb_dm__highestsensitivelevel', 'ocm', 'oc_oncokb_dm__all', 'seqq', 'nlod', 'contq', 'nalod', 'oc_base__note', 'oc_cancer_hotspots__samples', 'oc_oncokb_dm__highestresistancelevel', 'oc_oncokb_dm__tumorsummary', 'oc_oncokb_dm__highestdiagnosticimplicationlevel', 'oc_hess_drivers__signature', 'oc_hess_drivers__is_driver']
-        Int n_rows=300000
+        Int n_rows=100000
 
-        String docker_image="python"
+        String docker_image="depmapomics:test"
```

Comment:
This is not the image name that I think we should actually use. This should probably be `us.gcr.io/cds-docker-containers/depmapomics:master` and I think this should be changed to be a parameter to make it easier to test other branches.